### PR TITLE
chore(deps): RHINENG-8625 update ruby to 3.3

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: app/graphql/resolvers/concerns/collection.rb [hunk 1]
```diff
@@ -20,8 +20,8 @@ module Collection
         end
       end
 
-      def resolve(**kwargs)
-        filters = filter_list(**kwargs)
+      def resolve(**)
```
Comment (vkrizan) [inline app/graphql/resolvers/concerns/collection.rb:23]: What a lovely shortcut.

**Review by vkrizan — APPROVED** [review]

File: app/graphql/resolvers/concerns/collection.rb (hunk 1 above)
Comment (skateman) [reply app/graphql/resolvers/concerns/collection.rb:23]: You don't like it? :trollface:

Comment (vkrizan) [reply app/graphql/resolvers/concerns/collection.rb:23]: Quite the opposite. I've meant what I've said. :slightly_smiling_face:

Comment (skateman) [reply app/graphql/resolvers/concerns/collection.rb:23]: Wait for the other goodies we will have with 3.3 😉

**Pull request merged**